CONTRIBUTING.md

```diff
@@ -168,32 +185,72 @@ The Core Contributor group includes:
 
-Core Contributors attend all project meetings and calls.
+Core Team attend all project meetings and calls. Allowances will be made for timezones and other commitments.
 
-#### GitHub Organisation Members
+#### Members Team
 
-GitHub Organisation Members are well-known contributors with a track record of:
+The Members Team are contributors who are well-known to the community with a track record of:
 
-* Fixing, testing and triaging issues
-* Joining contributor meetings and supporting new contributors
-* Testing and reviewing pull requests
-* Offering other project support, feedback and being available to help
+* fixing, testing and triaging issues and PRs
+* offering support to the project
+* providing feedback and being available to help where needed
+* testing and reviewing pull requests
+* joining contributor meetings and supporting new contributors
 
-Varying levels of write access are made available via the project bot [Derek](https://github.com/alexellis/derek) to help regular contributors transition to GitHub Organisation Membership. Members gain access to a private Slack channel and are also featured on the Team page of the OpenFaaS website.
+> Note: An essential skill for being in a team is communication. If you cannot communicate with your team on a regular basis, then membership may not be for you and you are welcome to contribute as community.
 
-GitHub Organisation Members are expected to:
+Varying levels of write access are made available via the project bot [Derek](https://github.com/alexellis/derek) to help regular contributors transition to the Members Team.
 
+Members Team Perks:
+* access to a private Slack channel
+* profile posted on the Team page of the OpenFaaS website
+* membership of the GitHub organisations openfaas/openfaas-incubator
+
+Upon request and subject to availability:
+* 1:1 coaching & mentorship
+* help with speaking opportunities and CfP submissions
+* help with CV, resume and LinkedIn profile
+* review, and promotion of blogs and tutorials on social media
+
+The Members Team are expected to:
+
+* participate in the members channel and engage with the topics
 * participate in community Zoom calls (when possible within your timezone)
 * make regular contributions to the project codebase
-* participate in the members channel and engage with the topics
 * take an active role in the public channels: #contributors and #openfaas
 * comment on and engage with project proposals
+* attend occasional 1:1 meetings with members of the Core Team or the Project Lead
+
+This group is intended to be an active team that shares the load and collaborates together. This means engaging in topics on Slack, encouraging other teammates, sharing ideas, helping the users and raising issues with the Core Team.
+
+The Members Team will notify the Members Team of any leave of a week or more and set a status in Slack of "away".
```

Comment:
> The Members Team will notify the Members Team...

Sounds a bit redundant. Perhaps `Members will notify the rest of the Members Team...`

Reply:
you're right. thank you